acf-recaptcha-v5.php

```diff
@@ -65,20 +69,6 @@ function render_field_settings($field) {
             'new_lines' => false
         ));
 
-        acf_render_field_setting($field, array(
-            'label' => __('Site Key', 'acf-recaptcha'),
-            'instructions' => __('Enter your site key from Google reCAPTCHA.', 'acf-recaptcha'),
-            'name' => 'site_key',
-            'required' => true,
-        ));
-
-        acf_render_field_setting($field, array(
-            'label' => __('Secret Key', 'acf-recaptcha'),
-            'instructions' => __('Enter your secret key from Google reCAPTCHA.', 'acf-recaptcha'),
-            'name' => 'secret_key',
-            'required' => true,
-        ));
-
```

Comment:
As mentioned, it would be a good idea to keep the settings for the individual fields.

Reply:
Also add a note in the instructions that it defaults to the keys specified in the main settings page.